### PR TITLE
feat: support for time series feature extraction

### PR DESCRIPTION
**Pull request by m-muecke**
Closes: https://github.com/mlr-org/mlr3fda/issues/140

@sebffischer couple of questions:

1. how do we handle `...` for learners i.e. do we pass `...` to the learners and add it to the param set?
2. check if you're fine with the naming of the PipeOp
3. where is the check for the package installation of the learner normally done, I've done it in the `.transform_dt` method or do you already do it in the constructor of the class

### CODE REVIEW

File: R/PipeOpFDATSFeatures.R [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        packages = c("mlr3fda", "mlr3pipelines", "tf", "tsfeatures"),
+        feature_types = c("tfd_reg", "tfd_irreg"),
+        tags = "fda"
+      )
+    }
+  ),
+
+  private = list(
+    .transform_dt = function(dt, levels) {
+      require_namespaces(
```
Comment (sebffischer) [inline R/PipeOpFDATSFeatures.R:89]: This should not be necessary when `"tsfeatures"` is added to `packages`

File: tests/testthat/test_PipeOpFDATSFeatures.R [hunk 2]
```diff
@@ -0,0 +1,22 @@
+test_that("PipeOpFDATSFeatures - basic properties", {
```
Comment (sebffischer) [inline tests/testthat/test_PipeOpFDATSFeatures.R:1]: add some more tests

**Review by sebffischer — CHANGES_REQUESTED** [review]

File: R/PipeOpFDATSFeatures.R (hunk 1 above)
Comment (m-muecke) [reply R/PipeOpFDATSFeatures.R:89]: But I've put it under suggests, then I would have to direct put it in imports

Comment (sebffischer) [reply R/PipeOpFDATSFeatures.R:89]: I don't fully understand your point

Comment (m-muecke) [reply R/PipeOpFDATSFeatures.R:89]: Simply that I don't want to put it in imports, but then its already being checked if its installed.

**Comment by sebffischer** [comment]
@m-muecke `...` are not really supported in paradox.

**Review by sebffischer — APPROVED** [review]

**Pull request merged**